clients/python/src/myanmartools/__init__.py

```diff
@@ -1,29 +1,5 @@
-'''
-Myanmar Tools
```

Comment:
Why did you remove the docs from this file?

Reply:
The package docstring is used when the user types `help(myanmartools)`. The first line in the docstring (the summary) is shown in the `NAME` section, and if the second line is a blank line, then the lines starting from the third line (the description) are shown in the `DESCRIPTION` section. The docstring for `ZawgyiDetector` is also shown in the `CLASSES` section since it is imported at the package level. So I think it is clear to just provide the summary line. Usage examples and tutorials should not be put in the package docstring. Besides, they are already in `README.rst`. If the package grows very large, we can use `sphinx` to create documentation and include one or more tutorial sections.

By the way, I used `flake8` (including `flake8-docstrings`) to check the `src` and `tests` directories.